gigadb/app/tools/wasabi-migration/rclone_copy.sh

```diff
@@ -102,10 +102,10 @@ fi
 if [ "$starting_doi" -lt 101000 ];
 then
     dir_range="100001_101000"
-elif [ "$starting_doi" -lt 102000 ] && [ "$starting_doi" -gt 101001 ];
+elif [ "$starting_doi" -lt 102001 ] && [ "$starting_doi" -gt 101000 ];
 then
     dir_range="101001_102000"
-elif [ "$starting_doi" -lt 103000 ] && [ "$starting_doi" -gt 102001 ];
+elif [ "$starting_doi" -lt 103001 ] && [ "$starting_doi" -gt 102000 ];
```

Comment:
Hi @pli888,

In this loop, it seems failed to handle if the starting_doi equals to `101000`, which would return empty value.

We could make use of `le` and `ge` operators, more details at [here](https://www.tutorialspoint.com/unix/unix-basic-operators.htm). 

The following sample code could cover all the ranges:
```
if [ "$starting_doi" -le 101000 ];
then
    dir_range="100001_101000"
    echo $dir_range
elif [ "$starting_doi" -le 102000 ] && [ "$starting_doi" -ge 101001 ];
then
    dir_range="101001_102000"
    echo $dir_range
elif [ "$starting_doi" -le 103000 ] && [ "$starting_doi" -ge 102001 ];
then
    dir_range="102001_103000"
    echo $dir_range
fi
```

Reply:
Hi @kencho51, @pli888 
In this context, I agree with @kencho51's idea. Without seeing is comment I was about to write the same thing.
A nice bash cheatsheet for conditional operators can be found here: https://devhints.io/bash#conditionals.

Something else, less critical,  I want to point out: the `then` expression should be on the same line as the `if` or `elif`